cristin-import/src/test/java/no/unit/nva/cristin/mapper/CristinMapperTest.java

```diff
@@ -401,23 +400,59 @@ public void mapThrowsMissingFieldsExceptionWhenNonIgnoredFieldIsMissing() {
         mapSetsNameToNullWhenBothFamilyNameAndGivenNameAreMissing();
     }
 
+    @ParameterizedTest(name = "map returns journal with DOI URI when input is Journal with DOI:{0}")
```

Comment:
Tests for ensuring that the DoiValidator is applied when we are receiving a DOI value from Cristin.